parameters/dict_parameter.py

```diff
@@ -36,7 +35,7 @@ def __getitem__(self, key):
 
     def keys(self):
         """Helper method to make DictParameter behave like dict"""
-        return self._value.keys()
+        return list(self._value.keys())
```

Comment:
New syntax? I thought it's already a list.

Reply:
Not anymore in python 3:

```
etienne@:~/Downloads $ python3.6
Python 3.6.3 (default, Oct  5 2017, 10:19:42) 
[GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.37)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> my_dict = {1:'a', 2:'b'}
>>> my_dict.keys()
dict_keys([1, 2])
>>> list(my_dict.keys())
[1, 2]
>>> 
```

The `dict_keys` object is still iterable. So it's ok in a for loop:
```
for key in my_dict.keys():
```

Reply:
Ah I see... it's now a `<class 'dict_keys'>` in python 3.6